### PR TITLE
BAEL-5123: Annotation Processor for Hibernate Validator

### PR DESCRIPTION
**Pull request by exaucae**
See http://jira.baeldung.com/browse/BAEL-5123.

### CODE REVIEW

**Comment by ashleyfrieze** [comment]
Retest this please

File: javaxval/intelliJ-formatter.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true"/>
+        <option name="TERNARY_OPERATION_WRAP" value="5"/>
+        <option name="ARRAY_INITIALIZER_WRAP" value="1"/>
+        <option name="VARIABLE_ANNOTATION_WRAP" value="2"/>
+        <indentOptions>
+            <option name="CONTINUATION_INDENT_SIZE" value="2"/>
+            <option name="SMART_TABS" value="true"/>
+        </indentOptions>
+    </codeStyleSettings>
+</code_scheme>
```
Comment (ashleyfrieze) [inline javaxval/intelliJ-formatter.xml:39]: Do we need this file adding to this module?

File: javaxval/pom.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (ashleyfrieze) [inline javaxval/pom.xml:4]: Revert this formatting change

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: javaxval/intelliJ-formatter.xml (hunk 1 above)
Comment (exaucae) [reply javaxval/intelliJ-formatter.xml:39]: No, I'll remove it.

File: javaxval/pom.xml (hunk 2 above)
Comment (ashleyfrieze) [reply javaxval/pom.xml:4]: This change is still not reverted

Comment (exaucae) [reply javaxval/pom.xml]: [It has, now](https://github.com/eugenp/tutorials/pull/11673/commits/0450f0cc4361cf8e0cf673a46aea5b1fe0bf8097) @ashleyfrieze .

**Pull request merged**